src/config/relay-chains.tsx

```diff
@@ -41,6 +43,7 @@ type CollateralToken = Currency<CollateralUnit>;
 type GovernanceToken = Currency<CurrencyUnit>
 
 let APP_NAME: string;
+let CROWDLOAN_DOMAIN: string;
```

Comment:
@tomjeatt 
Minor: this could be named `CROWDLOAN_LINK` in a more harmonized way.